src/gnn_reco/models/task/reconstruction.py

```diff
@@ -62,7 +62,7 @@ class EnergyReconstruction(Task):
 
     def _forward(self, x):
         # Transform energy
-        return torch.pow(10, x[:,0] + 1.).unsqueeze(1)
+        return x[:,0].unsqueeze(1) #torch.pow(10, x[:,0] + 1.).unsqueeze(1)
```

Comment:
I think we should aim to have each task return the natural, physical quantity — here: energy — rather than any transformed quantity that the user then needs to (manually) inverse-transform. I think that any scaling of physical quantities for the purposes of training alone should be a matter for the loss function. See comment above to `LogCoshLoss` along with the proposed resolution.